src/components/flowchart/flowchart.test.js

```diff
@@ -245,6 +246,50 @@ describe('FlowChart', () => {
     expect(wrapper.render().find('.pipeline-node--active').length).toBe(2);
   });
 
+  it('applies collapsed-hint class to nodes with input parameters are hovered during collapsed state', () => {
+    const mockNodes = getNodeIDs(mockState.animals);
+    const wrapper = setup.mount(
+      <FlowChart
+        hoveredParameters={true}
+        nodeTypeDisabled={{ parameters: true }}
+        nodesWithInputParams={{
+          [mockNodes[0]]: true,
+          [mockNodes[1]]: true,
+        }}
+      />
+    );
+    expect(wrapper.render().find('.pipeline-node--collapsed-hint').length).toBe(
+      2
+    );
+  });
+
+  it('applies parameter-indicator--visible class to nodes with input parameters when nodeDisabled prop set', () => {
+    const mockNodes = getNodeIDs(mockState.animals);
+    const wrapper = setup.mount(
+      <FlowChart
+        nodeTypeDisabled={{ parameters: true }}
+        nodesWithInputParams={{
+          [mockNodes[0]]: true,
+          [mockNodes[1]]: true,
+        }}
+      />
+    );
+    expect(
+      wrapper.render().find('.pipeline-node__parameter-indicator--visible')
+        .length
+    ).toBe(2);
+  });
+
+  it('applies .parameters class to all parameter nodes', () => {
+    const wrapper = setup.mount(<FlowChart />);
+    expect(wrapper.render().find('.pipeline-node--parameters').length).toBe(4);
+  });
+
+  it('applies .parameters class to all the edges from parameter nodes', () => {
+    const wrapper = setup.mount(<FlowChart />);
+    expect(wrapper.render().find('.pipeline-edge--parameters ').length).toBe(4);
```

Comment:
Not a big thing but probably should remove the trailing space on the selector here otherwise it might appear something missing

Reply:
Woah, eye of an eagle @bru5